commander/src/commands/core/install.ts

```diff
@@ -255,7 +256,11 @@ export default class InstallCommand extends BaseCommand {
 								);
 
 								createDirectory(installDir);
-								await extract(fileDir, fileName, installDir);
+								await tar.x({
```

Comment:
This is not really related, is it?

Reply:
i just removed the old function and now we can use the npm package for the extract. so all good